.github/workflows/linting.yml

```diff
@@ -28,14 +28,10 @@ jobs:
       - name: List dependencies
         run: pip freeze
 
-      - name: Lint with isort
+      - name: Ruff src
         if: ${{ always() }}
-        run: isort --check-only src tests
+        run: ruff check src 
 
-      - name: Lint with black
+      - name: Ruff tests
         if: ${{ always() }}
-        run: black src tests
-
-      - name: Lint with flake8
-        if: ${{ always() }}
-        run: flake8 src tests
+        run: ruff check src
```

Comment:
Just merge into `Ruff check` on `ruff check src tests`? Second one does not actually check tests